frontend/src/metabase/querying/components/FilterModal/FilterModal.tsx

```diff
@@ -1,23 +1,23 @@
 import { useMemo, useState } from "react";
 import { t } from "ttag";
-import { Button, Flex, Modal, Tabs } from "metabase/ui";
+import { Button, Flex, Modal, Stack, Tabs, Text } from "metabase/ui";
```

Comment:
For me the vertical scrollbar is always shown in the modal:

![image](https://github.com/metabase/metabase/assets/6830683/0166abc0-ed63-4ac1-be05-830d410819fc)